default/scripting/ship_hulls/organic/organic.macros

```diff
@@ -3,7 +3,7 @@ LIVING_HULL_EFFECTS_GROUPS
 '''EffectsGroup
     scope = Source
     effects = [
-        SetStructure value = Value + (2 * [[SHIP_STRUCTURE_FACTOR]])
+        SetStructure value = Value + (NamedInteger name = "SHP_LIVING_HULL_REPAIR" value = (2 * [[SHIP_STRUCTURE_FACTOR]]))
```

Comment:
should be a `NamedReal`, I think. Meter setting effects' `value` is a real (double-precision) number, so this will just require an extra typecast